azext_edge/edge/providers/support/akri.py

```diff
@@ -22,47 +22,62 @@
 logger = get_logger(__name__)
 
 
-AKRI_NAME_LABEL = "name in (aio-akri-agent)"
+AKRI_INSTANCE_LABEL = "app.kubernetes.io/instance in (akri)"
+AKRI_APP_LABEL = "app in (otel-collector)"
```

Comment:
the value for this label doesn't look for AKRI specific

Reply:
it grabs the akri otel collector and not the common one